src/main/scala/com/mpc/scalats/core/Compiler.scala

```diff
@@ -29,6 +32,14 @@ object Compiler {
   )(implicit config: Config): ListSet[Declaration] =
     compile(scalaTypes, Option.empty[InterfaceDeclaration])
 
+  implicit def typeNameToString(typeName: TypeName)(implicit config: Config): String = {
+    if (config.prependEnclosingClassNames) {
```

Comment:
I don't see why this is configurable, instead of being the built-in/default

Reply:
Hmm I don't think the plugin was designed with that being default in mind... esp. looking at the unit tests, all `(I)TestClasses` are nested within `ScalaFixtures` and yet this is not meant in unit tests at all. I'd rather keep it configurable TBH.